From b649a48d07f0b91f75ef21f5990abfa646094086 Mon Sep 17 00:00:00 2001
From: s0dyy <msorin@msorin.com>
Date: Sun, 8 Aug 2021 20:19:59 +0200
Subject: [PATCH] Use CMAKE_INSTALL_LIBDIR instead of /usr/lib/cmake/FishUI
Upstream: Yes and already merged, it can be removed in the next release.

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6b0fca0..71b0ece 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -66,7 +66,7 @@ if (MSVC)
     set(CMAKECONFIG_INSTALL_DIR "${_VCPKG_INSTALLED_DIR}/${VCPKG_TARGET_TRIPLET}/share/FishUI")
     set(CMAKE_INSTALL_PREFIX "${_VCPKG_INSTALLED_DIR}")
 else ()
-    set(CMAKECONFIG_INSTALL_DIR "/usr/lib/cmake/FishUI")
+  set(CMAKECONFIG_INSTALL_DIR "${CMAKE_INSTALL_LIBDIR}/cmake/FishUI")
 endif ()
 message(STATUS "install directory:" "${CMAKECONFIG_INSTALL_DIR}")
 
-- 
2.32.0

